Bundle/CoreBundle/Resources/views/Widget/Form/stylize/form.html.twig

```diff
@@ -0,0 +1,29 @@
+{% trans_default_domain "victoire" %}
+{% form_theme form 'VictoireFormBundle:Form:fields.html.twig' %}
+
+{{ form_start(form, {'attr' : {'class' : 'vic-form-active'}}) }}
+
+<div id="vic-stylize">
+
+    <h3 class="vic-h3">
+        {{ 'victoire.widget.stylize.global.heading'|trans() }}
+    </h3>
+    <div class="vic-row">
+        <div class="vic-col-sm-6 vic-col-md-4">{{ form_row(form.containerTag) }}</div>
+        <div class="vic-col-sm-6 vic-col-md-4">{{ form_row(form.containerClass) }}</div>
+        <div class="vic-col-sm-6 vic-col-md-4">{{ form_row(form.theme) }}</div>
+        <div class="vic-col-xs-12">{{ form_row(form.deleteBackground) }}</div>
```

Comment:
I do not like this principle, I'd rather to show the legacy field in a "deprecated section" with a deprecation message "This field is deprecated, it will be totally removed in version 1.2.0, please use the new background system instead."